sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -113,7 +113,7 @@ class RadarLocationReceiver : BroadcastReceiver() {
                     return
                 }
 
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP && !RadarForegroundService.started) {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && !RadarForegroundService.started) {
```

Comment:
Why are we increasing it from version requirement from 5.0 to 8.0? Thanks for the clarification :)

Reply:
Not sure how this didn't trigger any warnings previously but 
```
@RequiresApi(Build.VERSION_CODES.O)
class RadarForegroundService : Service() {
```